dev_apps/music-nga/views/albums/view.js

```diff
@@ -6,11 +6,12 @@ var debug = 1 ? (...args) => console.log('[AlbumsView]', ...args) : () => {};
 var AlbumsView = View.extend(function AlbumsView() {
   View.call(this); // super();
 
-  this.search = document.getElementById('search');
+  this.searchBox = document.getElementById('search');
```

Comment:
Again.. Clearly I lack capacity for having intelligent thoughts today. I see why you did this now.